graph/community/louvain_common.go

```diff
@@ -306,6 +308,36 @@ func (d *dense) TakeMin(p *int) bool {
 	return true
 }
 
+// slice is a sparse integer set iterator.
+type slice struct {
```

Comment:
The type reads as `community.slice` which does not communicate its use clearly. Perhaps change from `slice` to `intSetSlice`?